content/guides/component-testing/react/api.md

```diff
@@ -123,7 +123,10 @@ it('mounts', () => {
 
 ### MountReturn
 
+Type that the `mount` function returns
```

Comment:
Shouldn't this be yielded, not returned?

Reply:
Ah yeah, good catch - wrong verbiage. 

Just to clarify for anyone reading/wondering about this

```js
cy.mount(<Foo />).then(val => ...)
```

The `val` is *yielded*. `cy.mount` _does_ return something, a `Chainable` (so you can do `cy.mount(...).get(...)`). The yielded value is what you get in the `then`.

Reply:
Yeah, I copied existing verbiage from the Angular, command, will update both.

Reply:
cool, updated everywhere